circle.yml

```diff
@@ -1,4 +1,30 @@
-general:
-  branches:
-    only:
-      - /circle-.*/
+machine:
+  xcode:
+    version: 7.3
+  environment:
+    XCODE_PROJECT: build/tangram.xcodeproj
+    XCODE_SCHEME: phony # Dummy value, Circle won't run without a scheme.
+checkout:
+  post:
+    - git submodule sync
+    - git submodule update --init --recursive
+test:
+  pre:
+    - set -o pipefail &&
+      mkdir build &&
+      cd build &&
+      cmake ..
+      -DPLATFORM_TARGET=ios.framework
+      -DIOS_PLATFORM=SIMULATOR
+      -DCMAKE_TOOLCHAIN_FILE=toolchains/iOS.toolchain.cmake
```

Comment:
If you're targeting the ios.framework target, would you want iOS.framework.cmake here instead?